### PR TITLE
registrySyncer: No importing back

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

File: pkg/controller/registrysyncer/registrysyncer.go [hunk 1]
```diff
@@ -235,7 +235,8 @@ func (r *reconciler) reconcile(ctx context.Context, req reconcile.Request, log *
 		log.Debug("creating imageStreamImport")
 		if err := createAndCheckStatus(ctx, client, imageStreamImport); err != nil {
 			controllerutil.CountImportResult(ControllerName, clusterName, req.Namespace, imageStreamName, false)
-			return fmt.Errorf("failed to create and check the status for imageStreamImport on cluster %s %v : %w", clusterName, imageStreamImport, err)
+			//no return of the error to avoid staying in the queue forever
+			log.WithError(err).Errorf("failed to create and check the status for imageStreamImport for tag %s of %s in namespace %s on cluster %s", imageTag, imageStreamImport.Name, imageStreamImport.Namespace, clusterName)
```
Comment (alvaroaleman) [inline pkg/controller/registrysyncer/registrysyncer.go:239]: This kind of defeats the purpose of using a controller in the first place, as we will never retry on transient errors. Also, metrics are wrong

Comment (alvaroaleman) [reply pkg/controller/registrysyncer/registrysyncer.go:239]: I think what we should be doing is if the src import had any errors and skip the ones where it had errors

Comment (hongkailiu) [reply pkg/controller/registrysyncer/registrysyncer.go]: Implemented as discussed in slack: no importing back to where the image came from.

Comment (hongkailiu) [reply pkg/controller/registrysyncer/registrysyncer.go]: >metrics are wrong

Can you elaborate?

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**